utils/runtime-fuzzer/src/arbitrary_call.rs

```diff
@@ -42,9 +39,23 @@ static_assertions::const_assert!(MAX_PAYLOAD_SIZE <= gear_core::message::MAX_PAY
 ///
 /// It's main purpose is to be an implementor of `Arbitrary` for the array of [`GearCall`]s.
 /// New-type is required as array is always a foreign type.
-#[derive(Debug, Clone)]
+#[derive(Clone)]
 pub struct GearCalls(pub [GearCall; GearCalls::MAX_CALLS]);
 
+/// That's done because when fuzzer finds a crash it prints a [`Debug`] string of the [`GearCalls`].
+/// Fuzzer executes [`GearCalls`] with pretty large codes and payloads, therefore to avoid printing huge
+/// amount of data we do a mock implementation of [`Debug`].
+///
+/// If one wants to see a real debug string of the type, a separate wrapper over [`GearCall`]s array must be
+/// implemented. This wrapper must implement [`Debug`] then.
+impl Debug for GearCalls {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_tuple("GearCalls")
+            .field(&"Mock `Debug` impl")
```

Comment:
Should the `field` be `&&str`? Doesn't `&str` work?

Reply:
It expects `&dyn Debug`, so `&str` doesn't compile